FUNCTIONS.md

```diff
@@ -0,0 +1,21 @@
+# Presto Function Guidelines
+
+Presto includes a large library of built-in, core SQL functions such as AVG and CONCAT that are available in every deployment.
+
+Presto also includes extension functions that can optionally be enabled in particular deployments.
+
+Finally, Presto allows users to write user defined functions (UDFs) in either Java or SQL that are not stored in this GitHub repository.
+
+This document lays out some guidelines for which category is right for which functions. 
+
+Functions defined by the ANSI SQL spec should be written as core functions in this repository.
```

Comment:
```suggestion
Functions defined by the [ANSI SQL spec](https://jakewheat.github.io/sql-overview/) should be written as core functions in this repository.
```

Okay, that works for me given the context!

Reply:
done